simple/operator.go

```diff
@@ -0,0 +1,150 @@
+package simple
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/grafana/grafana-app-sdk/k8s"
+	"github.com/grafana/grafana-app-sdk/operator"
+	"github.com/grafana/grafana-app-sdk/resource"
+	"k8s.io/client-go/rest"
+)
+
+type WebhookConfig struct {
+	Enabled   bool
+	Port      int
+	TLSConfig k8s.TLSConfig
+}
```

Comment:
I would add validating and mutating controllers as part of this configuration. I don't see a reason why someone would enable webhooks but leave out controllers and vice versa. This would also simplify the code.